cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -12,6 +12,26 @@ WORKER_NODE_ROOT="${CLUSTER_NAME}-worker"
 
 OPERATOR_GIT_HASH=8d3c30de8ec5a9a0c9eeb84ea0aa16ba2395cd68  # release-4.4
 
+function ensure_k8s_object {
+  object_type=$1
```

Comment:
All these variables better be set as `local` with `-r` to make then read-only, otherwise they are shared all over.
Example: `local -r object_type=$1`.